vc/vc.go

```diff
@@ -33,6 +34,103 @@ func VCContextV1URI() ssi.URI {
 	}
 }
 
+const (
+	// JSONLDCredentialProofFormat is the format for JSON-LD based credentials.
+	JSONLDCredentialProofFormat string = "ldp_vc"
+	// JWTCredentialsProofFormat is the format for JWT based credentials.
+	JWTCredentialsProofFormat = "jwt_vc"
+)
+
+var errCredentialSubjectWithoutID = errors.New("credential subjects have no ID")
```

Comment:
`credentialSubject.id` is not required. Is this any different when jwt proof format is used?

Reply:
No, I misinterpreted the spec. I will treat it no differently.